StereoKit.HolographicRemoting/HolographicRemoting.cs

```diff
@@ -54,6 +56,21 @@ void OnPreCreateSession()
 			contextProperties.maxBitrateKbps              = (uint)_maxBitrate;
 			contextProperties.videoCodec                  = XrRemotingVideoCodecMSFT.H265;
 			contextProperties.depthBufferStreamResolution = XrRemotingDepthBufferStreamResolutionMSFT.HALF;
+
+			if (_sendAudio && !_wideAudioCapture)
+			{
+				XrRemotingAudioOutputCaptureSettingsMSFT audioCaptureSettings = new XrRemotingAudioOutputCaptureSettingsMSFT();
+				audioCaptureSettings.type                   = XrStructureType.REMOTING_AUDIO_OUTPUT_CAPTURE_SETTINGS_MSFT;
+				audioCaptureSettings.audioOutputCaptureMode = XrRemotingAudioOutputCaptureModeMSFT.AUDIO_OUTPUT_CAPTURE_MODE_APP_ONLY_MSFT;
+				
+				int     size    = Marshal.SizeOf(typeof(XrRemotingAudioOutputCaptureSettingsMSFT));
+				IntPtr  memory  = Marshal.AllocHGlobal(size);
+				Marshal.StructureToPtr(audioCaptureSettings, memory, false);
+
+				contextProperties.next = memory;
+				Marshal.FreeHGlobal(memory);
```

Comment:
You're freeing the memory right after assigning it into the .next chain here! This should be freed after `contextProperties` is used by `xrRemotingSetContextPropertiesMSFT`, otherwise the memory could be anything. I didn't _see_ any crashed related to this, but I expect that's purely by chance?